src/main/drivers/vtx_rtc6705.c

```diff
@@ -0,0 +1,247 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * Author: Giles Burgess (giles@multiflite.co.uk)
+ *
+ * This source code is provided as is and can be used/modified so long
+ * as this header is maintained with the file at all times.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "platform.h"
+
+#include "common/maths.h"
+
+#include "drivers/vtx_rtc6705.h"
+#include "drivers/bus_spi.h"
+#include "drivers/system.h"
+#include "drivers/gpio.h"
+
+#define RTC6705_SET_HEAD 0x3210 //fosc=8mhz r=400
+#define RTC6705_SET_A1 0x8F3031 //5865
```

Comment:
this code was ported from older PR's, there is also code for the rtc6705 via soft_spi floating around in another PR and in betaflight - the two code bases need further alighnment/de-duplication and cleanup - this will happen in a future exercise but let it be noted that I didn't like either of the two PR's but time pressures forced me to choose this code.